### DISCUSSION TITLE
전체 UI 수정사항 건의

### DISCUSSION THREAD

**Discussion by violet9503** [prgrms-web-devcourse/Team-10jo-10wonmoa-FE#125] [General]
- 전체 UI 수정사항 건의 (Ex. 색상, padding, border 등)

**Comment by violet9503** [comment]
TopNavTitle 글씨 크기 줄이기

**Comment by violet9503** [comment]
가계부, 검색 각 Item의 margin: 0 2rem 이 너무 큰 것 같아보임

**Comment by violet9503** [comment]
'${금액 포매팅}원' 까지 보여줄 것인지 ${금액 포매팅}까지만 보여줄 것인지 통일

**Reply by dahye1013** [reply]
원까지 보여주는게 맞을 것 같긴한데... 
액수가 크면 '...' 이런식으로 잘라야할 것 같은데 👀 
10,000,000원  => 1,000...원 
이렇게 보니까 좀 어색한것 같기도해요
